packages/scaffold-ui/src/views/w3m-profile-view/index.ts

```diff
@@ -143,7 +143,7 @@ export class W3mProfileView extends LitElement {
         : html`<wui-button
```

Comment:
I think we need to remove the following line in this page as well?
```js
AccountController.setShouldUpdateToAddress(account.address, ChainController.state.activeChain)
```

Reply:
why?

Reply:
if the user switches, UI still needs to update. This pr only removes intermediate screen on connection !